OWNERS

```diff
@@ -1,19 +1,17 @@
 # Reviewers can /lgtm /approve but not sufficient for auto-merge without an
 # approver
 reviewers:
-- Rajakavitha1
-- stewart-yu
-- xiangpengzhao
-- zhangxiaoyu-zidif
+# - kbkawkey
+- tengqm
 
 # Approvers have all the ability of reviewers but their /approve makes
 # auto-merge happen if a /lgtm exists, or vice versa, or they can do both
 # No need for approvers to also be listed as reviewers
 approvers:
-- bradamant3
-- bradtopol
+- jimangel
 - kbarnard10
-- steveperry-53
+# - kbhawkey
+# - onlydole
```

Comment:
@tengqm , Do you want to remove the pound signs?

Reply:
'kbhawkey', 'onlydole' and 'sftim' are not members of 'kubernetes-sigs', so I was getting "do-not-merge/invalid-owners-file" label for this PR.

Reply:
OK. Do you know how we become "members"?

Reply:
I've just dug into the issue. It looks like `https://github.com/kubernetes/org/blob/master/config/kubernetes-sigs/org.yaml#L632-L644` is the place to edit it. Will sync with @zacharysarah on the next step.

Reply:
@tengqm Thanks for raising visibility--you are correct, the file in k/org sets admin/write permissions for this repo.

@kbhawkey @onlydole @sftim If I understand correctly, you need to:

1. Open an issue in k/org to [apply for membership](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E) in the kubernetes-sigs org. Feel free to assign me and @tengqm for a +1.

2. Once you're approved as members of kubernetes-sigs, you should be add-able to `OWNERS` and `org.yaml`. 

Do you need write permissions to generate reference docs?

@tengqm Thanks for tracking down the issue. 🙇

Reply:
No, not to generate the docs.

Reply:
GitHub issue filed at: https://github.com/kubernetes/org/issues/2006